Code/Framework/AzToolsFramework/AzToolsFramework/UI/Prefab/LevelRootUiHandler.cpp

```diff
@@ -79,16 +85,76 @@ namespace AzToolsFramework
             return;
         }
 
-        QPen borderLinePen(s_levelRootBorderColor, s_levelRootBorderThickness);
+        // Draw border line at the bottom
+        QPen borderLinePen(m_levelRootBorderColor, PrefabUIConstants::levelDividerThickness);
         QRect rect = option.rect;
-
-        painter->save();
-        painter->setRenderHint(QPainter::Antialiasing, true);
-
-        // Draw border at the bottom
         painter->setPen(borderLinePen);
         painter->drawLine(rect.bottomLeft(), rect.bottomRight());
 
+        if (Prefab::IsPrefabOverridesUxEnabled())
+        {
+            // Draw capsule
+            const bool isFirstColumn = index.column() == EntityOutlinerListModel::ColumnName;
+            const bool isLastColumn = index.column() == EntityOutlinerListModel::ColumnLockToggle;
+
+            QColor backgroundColor = m_prefabCapsuleColor;
+            AZ::EntityId levelContainerEntityId = m_prefabPublicInterface->GetLevelInstanceContainerEntityId();
+
+            if (m_prefabFocusPublicInterface->IsOwningPrefabBeingFocused(levelContainerEntityId))
+            {
+                backgroundColor = m_prefabCapsuleEditColor;
+            }
+            else if (!(option.state & QStyle::State_Enabled))
+            {
+                backgroundColor = m_prefabCapsuleDisabledColor;
+            }
+
+            QPainterPath backgroundPath;
+            backgroundPath.setFillRule(Qt::WindingFill);
```

Comment:
Suggestion: Move backgroundPath further down to where it's used.

Reply:
Can just move it down few lines before `if (isFirstColumn || isLastColumn)`.